kernel/common/ezautolinkoperator.php

```diff
@@ -33,7 +40,12 @@ function namedParameterList()
                                             'default' => null ) );
     }
 
-    function formatUri( $url, $max )
+    /*!
+     Leave for backward compatibilty.
+     \static
+     \deprecated
+    */
```

Comment:
For all class methods you touch / change / modify (at least) in this class file please convert the function doxygen comments to phpdoc (the code standard for all new eZ Publish Legacy code). For all new methods please create new phpdoc function comments. This step is required.